jdaviz/app.py

```diff
@@ -625,8 +625,8 @@ def get_data_from_viewer(self, viewer_reference, data_label=None,
         for layer_state in viewer.state.layers:
             label = layer_state.layer.label
 
-            if hasattr(layer_state, 'layer') and \
-                    (data_label is None or label == data_label):
+            if (hasattr(layer_state, 'layer') and
+                    (data_label is None or label == data_label)):
```

Comment:
```suggestion
                (data_label is None or label == data_label)):
```

Reply:
Style recommendation is to indent second line of if statement more as not to confuse with the code that is nested under it. What is the motivation for this suggestion?

Reply:
I always mix up this style error. Please ignore.